pkg/nsx/services/common/types.go

```diff
@@ -46,7 +46,7 @@ var (
 	ResourceTypeSecurityPolicy = "SecurityPolicy"
 	ResourceTypeGroup          = "Group"
 	ResourceTypeRule           = "Rule"
-	ResourceTypeVPC            = "VPC"
+	ResourceTypeVPC            = "Vpc"
```

Comment:
It's good that we use ResourceTypeVpc as well to follow our naming convention.

Reply:
done